Makefile

```diff
@@ -20,6 +20,11 @@ updatedeps:
 install:
 	go install -ldflags '-X "main.version=dev build $(dev_build_version)"' ./...
 
+.PHONY: recreate-bindata
+recreate-bindata:
+	cd internal/resources/webform/; go run github.com/go-bindata/go-bindata/go-bindata -o bindata.go -pkg=webform `find . -name '*.html' -or -name '*.css' -or -name '*.png' -or -name '*.js' -or -name '*.svg'`; gofmt -w -s bindata.go
```

Comment:
These should all be defined via `go:generate` directives, so the only thing the Makefile has to do is run `go generate ./...`.